app/views/dashboard/show.html.erb

```diff
@@ -2,15 +2,23 @@
   <div class="govuk-grid-column-full">
     <h1 class="govuk-heading-l">User dashboard</h1>
     <p class="govuk-body"> You are logged in</p>
-
     <% if @current_user.early_career_teacher? %>
       <h2 class="govuk-heading-l">Your Core Induction Programme</h2>
-      <% if @current_user.course_years.each do |course_year| %>
+      <% @current_user.course_years.each do |course_year| %>
         <p><%= govuk_link_to course_year.title, cip_year_url(course_year) %></p>
-      <% end.empty? %>
+      <% end %>
+      <% if @current_user.course_years.empty? %>
```

Comment:
Pity we are changing that, I quite liked the solution.

But vscode code formatting is too much of a pain in the ass to tell you that not all ends are created equal.